### PR TITLE
elprep/filter: fix output glob

### PR DESCRIPTION
**Pull request by matthdsm**
This PR fixes the output glob for `elprep filter` to avoid globbing the input data into the outputs

## PR checklist
- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/elprep/filter/main.nf [hunk 1]
```diff
@@ -65,7 +65,7 @@ process ELPREP_FILTER {
     def assembly_regions_cmd = get_assembly_regions ? " --assembly-regions ${prefix}.assembly_regions.igv": ""
 
     """
-    elprep filter ${bam} ${prefix}.${suffix} \\
+    elprep filter ${bam} output/${prefix}.${suffix} \\
```
Comment (jfy133) [inline modules/elprep/filter/main.nf:68]: Where is `output/` defined here?

Comment (matthdsm) [reply modules/elprep/filter/main.nf:68]: nowhere, I needed to define a random output directory name to avoid the collision with the input bam

Comment (matthdsm) [reply modules/elprep/filter/main.nf:68]: so the bam file is written to `output/test.bam`

Comment (jfy133) [reply modules/elprep/filter/main.nf:68]: Sorry completely missed in input ${bam} because I'm too used to one line per param/argument :facepalm:

**Review by jfy133 — APPROVED** [review]

File: modules/elprep/filter/main.nf (hunk 1 above)
Comment (matthdsm) [reply modules/elprep/filter/main.nf:68]: No problem, perhaps I should split it up into lines next time 😄

**Comment by matthdsm** [comment]
thanks for the ✅ !

**Pull request merged**